test/e2e/policies.test-suite/balloons/n4c16/test03-reserved/balloons-reserved.cfg

```diff
@@ -3,6 +3,8 @@ policy:
   ReservedResources:
     CPU: cpuset:0-2
   balloons:
+    PinCPU: true
+    PinMemory: true
```

Comment:
Can I ask why these are not the defaults ? I mean, isn't this exactly what the user us expected to use in 99% of the cases ? If it is, I think these should be opt-out, not opt-in.

Reply:
Hmm, yes, good question @klihub, that same question came to my mind as well 😎